pkg/storaged/client.js

```diff
@@ -410,14 +410,62 @@
                 });
         }
 
+        function query_fsys_info() {
+            var info = {
+                xfs: {
+                    can_shrink: false,
+                    can_grow: true,
+                    grow_needs_unmount: false
+                },
+
+                ext4: {
+                    can_shrink: true,
+                    shrink_needs_unmount: true,
+                    can_grow: true,
+                    grow_needs_unmount: false
+                },
```

Comment:
I figure this is a fallback for old udisks that don't support `CanResize()`. However, should we even show these buttons for old udisks versions? If they don't have CanResize, do these have the methods to actually *do* the resizing?

xfs is missing `shrink_needs_unmount`.

Reply:
> If they don't have CanResize, do these have the methods to actually do the resizing?

Yes, there is the `fsys_resize` flag for `LogicalVolume.Resize`.  That flag controls the `-r` option of `lvresize` which in turn calls `fsadm`.

Reply:
Ah, thanks. Likewise, `shrink_needs_unmount` doesn't get added as `can_shrink` is false, so this looks good.